bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -433,24 +461,25 @@ async function updateOrResendEnvelope({ client, dataStore, integrationParameters
 
 async function sendTemplateThreeRecipient({ client, dataStore, integrationParameters, actionParameters }) {
     const ACCOUNT_ID = integrationParameters.account_Id;
+    console.log(JSON.stringify(actionParameters))
     let templateRoles = [{
         roleName: actionParameters.role_Name_one,
         name: actionParameters.recipient_Name_one,
-        email: actionParameters.recipient_Email_one ?? actionParameters.recipient_Email_one_Unknown
+        email: actionParameters.recipient_Email_one != "" && actionParameters.recipient_Email_one != undefined ?  actionParameters.recipient_Email_one : actionParameters.recipient_Email_one_Unknown,
```

Comment:
It is a good habit to use triple equals.

https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/